domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterModule.kt

```diff
@@ -103,31 +113,39 @@ class PlatformParameterModule {
   fun provideSyncUpWorkerTimePeriod(
     platformParameterSingleton: PlatformParameterSingleton
   ): PlatformParameterValue<Int> {
-    return platformParameterSingleton.getIntegerPlatformParameter(
+    return overriddenPlatformParameters[
+      PlatformParameter.SYNC_UP_WORKER_TIME_PERIOD_IN_HOURS
+    ]?.let {
+      PlatformParameterValue.createDefaultParameter(it as Int)
+    } ?: platformParameterSingleton.getIntegerPlatformParameter(
       SYNC_UP_WORKER_TIME_PERIOD_IN_HOURS
-    ) ?: PlatformParameterValue.createDefaultParameter(
-      SYNC_UP_WORKER_TIME_PERIOD_IN_HOURS_DEFAULT_VALUE
     )
+      ?: PlatformParameterValue.createDefaultParameter(
+        SYNC_UP_WORKER_TIME_PERIOD_IN_HOURS_DEFAULT_VALUE
+      )
   }
 
   @Provides
   @EnableEditAccountsOptionsUi
   fun provideEnableEditAccountsOptionsUi(
     platformParameterSingleton: PlatformParameterSingleton
   ): PlatformParameterValue<Boolean> {
-    return platformParameterSingleton.getBooleanPlatformParameter(
-      EDIT_ACCOUNTS_OPTIONS_UI
-    ) ?: PlatformParameterValue.createDefaultParameter(
-      ENABLE_EDIT_ACCOUNTS_OPTIONS_UI_DEFAULT_VALUE
-    )
+    return overriddenFeatureFlags[FeatureFlag.EDIT_ACCOUNTS_OPTIONS_UI]?.let {
+      PlatformParameterValue.createDefaultParameter(it)
+    } ?: platformParameterSingleton.getBooleanPlatformParameter(EDIT_ACCOUNTS_OPTIONS_UI)
+      ?: PlatformParameterValue.createDefaultParameter(
+        ENABLE_EDIT_ACCOUNTS_OPTIONS_UI_DEFAULT_VALUE
+      )
```

Comment:
I don't understand why this is a combination of feature flag and platform parameter here and below.